.github/workflows/wheels-manylinux.yml

```diff
@@ -0,0 +1,257 @@
+        required: false
+        type: string
+        default: "auditwheel --verbose repair -w {dest_dir} --plat manylinux_2_17_x86_64 {wheel}"
+      auditwheel-repair-command-arm64:
+        required: false
+        type: string
+        default: "auditwheel --verbose repair -w {dest_dir} --plat manylinux_2_31_aarch64 {wheel}"
+      cibw-environment:
+        type: string
+        required: false
+        default: ''
+      cibw-before-all-amd64:
+        required: false
+        type: string
+        default: 'true'
+      cibw-before-all-arm64:
+        required: false
+        type: string
+        default: 'true'
+      cibw-before-build:
+        required: false
+        type: string
+        default: 'true'
+
+      skbuild-configure-options:
+        required: false
+        type: string
+        default: ''
+      skbuild-build-options:
+        required: false
+        type: string
+        default: ''
+
+      # test settings
+      test-container-amd64:
+        required: false
+        type: string
+        default: "rapidsai/manylinux_2_27"
+      test-container-arm64:
+        required: false
+        type: string
+        default: "rapidsai/manylinux_2_31"
+      test-docker-options:
+        required: false
+        type: string
+        default: '-e _NOOP'
+      test-checkout-src: # allow one to skip checking out source code to run packaged tests
+        required: false
+        type: string
+        default: 'true'
+      test-requires:
+        required: false
+        type: string
+        default: ''
+      test-extras:
+        required: false
+        type: string
+        default: ''
+      test-unittest: # tests are allowed to be blank because the wheel is installed and pip checked
+        required: false
+        type: string
+        default: ''
+      test-before-amd64:
+        required: false
+        type: string
+        default: 'true'
+      test-before-arm64:
+        required: false
+        type: string
+        default: 'true'
+
+jobs:
+  wheel-build:
+    name: cibuildwheel ${{ matrix.cpu-arch }} ${{ matrix.python-version }}
+    runs-on: ${{ matrix.labels }}
+    strategy:
+      matrix:
+        include:
+          - labels: [self-hosted, linux, amd64, cpu16]
+            cpu-arch: x86_64
+            python-version: 3.8
+          - labels: [self-hosted, linux, arm64, cpu16]
+            cpu-arch: aarch64
+            python-version: 3.8
+          - labels: [self-hosted, linux, amd64, cpu16]
+            cpu-arch: x86_64
+            python-version: 3.9
+          - labels: [self-hosted, linux, arm64, cpu16]
+            cpu-arch: aarch64
+            python-version: 3.9
+    steps:
+    - name: checkout code repo
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: 0 # unshallow fetch for setuptools-scm
+        persist-credentials: false
+
+    - name: Build wheels with custom RAPIDS cibuildwheel action
```

Comment:
Already discussed arch-specific cibuildwheel/actions.yml to pass through CIBW vars directly (where possible).